bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup.Tests/KeyCeremony/TestElectionPolynomial.cs

```diff
@@ -0,0 +1,87 @@
+namespace ElectionGuard.ElectionSetup.Tests.KeyCeremony;
+
+public class TestElectionPolynomial
+{
+    [Test]
+    public void Test_ElectionPolynomial_Constructs_WithDegrees()
+    {
+        // Arrange
+        var degrees = 2;
+
+        // Act
+        var result = new ElectionPolynomial(degrees);
+
+        // Assert
+        Assert.That(result, Is.Not.Null);
```

Comment:
Most of the patterns that we were trying to use for C# testing was not using `Assert.That` and would use `Assert.Equal` or something equivalent.

Reply:
https://github.com/microsoft/electionguard-core2/issues/188